tsql/TSqlParser.g4

```diff
@@ -261,49 +315,69 @@ client_assembly_specifier
     | STRING
     ;
 
-//Need to implement assembly_bits still
-assembly_bits:
-    ;
-
 assembly_option
     : PERMISSION_SET EQUAL (SAFE|EXTERNAL_ACCESS|UNSAFE)
     | VISIBILITY EQUAL (ON | OFF)
     | UNCHECKED DATA
-    | COMMA assembly_option
+    | assembly_option COMMA
     ;    
 
 network_file_share
-    : DOUBLE_BACK_SLASH computer_name=id file_path
+    : network_file_start network_computer file_path
+    ;
+ 
+network_computer
+    : computer_name=id
+    ;
+
+network_file_start
+    : DOUBLE_BACK_SLASH
     ;
 
 file_path
-    : '\\' file_path
+    : file_directory_path_separator file_path
     | id
     ;
 
+file_directory_path_separator
+    : '\\'
+    ;
+
 local_file
-    : DISK_DRIVE file_path
+    : local_drive file_path
     ;
 
+local_drive
+    :
+    DISK_DRIVE
+    ;
```

Comment:
Incorrect formatting. Use the following instead (the same for other such rules):
```ANTLR
    : DISK_DRIVE
    ;
```